index.html

```diff
@@ -0,0 +1,162 @@
+            <hr class="partner-underline">
+            <div class="desk-partners"><img class="img-fluid" src="images/images/partners_logo.png" alt=""></div>
+        </div>
+     </footer class="desk-footer">
+     </div>
+     <!-- End of Deskyop footer -->
+    <button class="see-more">MORE</button>
+    <button class="see-less">See Less</button>
+    <section  id="about-section">
+        <h2 class="hello about-hello">"Hello! Sharing world"</h2>
+        <h1 class="heading heading2">Youth Leadership <br> Summit 2023 </h1>
+        <p class="paragraph-2">The Global Summit brings together the community of  
+            experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+            This year, we hope to expand our invitation list including organizations and individuals.</p>
+            <p class="desk-about-par">The Global Summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+                This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that advance the cause of the Commons, free culture and open knowledge.
+                So if you're active and engaged in the worlds of open content and knowledge - free software advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together.</p>
+            <div class="contact-us">Please contact us per Email for any further questions about The Youth Submit 2023!</div>
+            <div class="email">youthsummit2023@gmail.com</div>
+    </section>
+    <div class="logo-explanation">Youth Leadership Summit 2023 Logo</div>
+         <hr class="hr1 hr21">
+         <div class="explanation">The logo of Youth Leadership Summit 2023 was decided through the logo competition from 8, June to 7, July.</div>
+         <p class="explanation2">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.
+            After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+            After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:</p>
+         <div class="summit"><img src="images/images/about_logo 1.png" alt=""></div>
+         <div class="summit-logo"><img class="img-fluid" src="images/images/about_logo 1 (1).png" alt=""></div>
+         <hr class="hr3 hr33">
+         <hr class="hr3">
+         <div class="past-summits">See the past Youth Summits</div>
+         <hr class="hr101">
+
+         <div class="look-in-past">Take a look at the past two Youth summits which took place in Moscow and Canada.</div>
```

Comment:
> You should build only these 2 pages:
The home page.
The about page.

- [ ] According to the project requirement, you are meant to have 2 pages, `About and Home page`. Currently, you only have one and it seems you also added some of the content of the about page in the home page. That's not how it should be. You are meant to create another HTML file `about.html` and implement the content t of the about page there. Also, remember to add the link to the about page in the nav list in your main navigation. :wave: :smile: 

- - Screenshot :+1: 
![Screenshot from 2023-08-10 10-10-32](https://github.com/denigit5/First-Capstone/assets/107632780/baec4005-26d4-44f5-b31e-8454e786a1d1)

Reply:
As requested by the previous code reviewer this is a required change and it need to be worked on. It would be nice if you corrected this inorder for you to get an approval. Please pay attention to detail i would advise you carefully go through the project requirements to avoid making mistakes. I would also encourage you to implement these changes to avoid wasting review chances.